src/calendar/day/basic/index.js

```diff
@@ -92,26 +93,29 @@ class Day extends Component {
     }
 
     return (
-      <TouchableOpacity
-        testID={this.props.testID}
-        style={containerStyle}
-        onPress={this.onDayPress}
-        onLongPress={this.onDayLongPress}
-        activeOpacity={activeOpacity}
-        disabled={shouldDisableTouchEvent}
-        accessibilityRole={isDisabled ? undefined : 'button'}
-        accessibilityLabel={this.props.accessibilityLabel}
-      >
-        <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
-        <Dot
-          theme={theme}
-          isMarked={marked}
-          dotColor={dotColor}
-          isSelected={selected}
-          isToday={isToday}
-          isDisabled={isDisabled}
-        />
-      </TouchableOpacity>
+      <View style={this.style.dayContainer}>
+        <TouchableOpacity
+          testID={this.props.testID}
+          style={containerStyle}
+          onPress={this.onDayPress}
+          onLongPress={this.onDayLongPress}
+          activeOpacity={activeOpacity}
+          disabled={shouldDisableTouchEvent}
+          accessibilityRole={isDisabled ? undefined : 'button'}
+          accessibilityLabel={this.props.accessibilityLabel}
+        >
+          <Text allowFontScaling={false} style={textStyle}>{String(this.props.children)}</Text>
+          <Dot
+            theme={theme}
+            isMarked={marked}
+            dotColor={dotColor}
+            isSelected={selected}
+            isToday={isToday}
+            isDisabled={isDisabled}
+          />
+        </TouchableOpacity>
+        {!!renderUnderDayView && renderUnderDayView(date)}
```

Comment:
I think you should move `renderUnderDayView` inside TouchableOpacity. It makes more sense to me to have this entire component touchable.

Reply:
> I think you should move `renderUnderDayView` inside TouchableOpacity. It makes more sense to me to have this entire component touchable.

I also think put into TouchableOpacity, but this is not suitable here, cuz the underView may have different event for each icons or may not wish clickable.

In some case the day underView may very huge.